vue/components/ui/templates/error-part/error-part.vue

```diff
@@ -138,6 +152,7 @@ export const ErrorPart = {
     methods: {
         goHome() {
             if (!this.navigation) return;
+            if (window) window.location = "/";

```

Comment:
use the router to push to "index"

Reply:
i tried this at first but it aparently it doesn't work in nuxt. I think it might be cause of the folder structure we have, it tries to pick a index/index.vue file and fails to find it.

we have no path or name for index so it looks as nothing will make router redirect to home, neither { path: "/" } works properly. This is why I opted for this pure js solution

Reply:
Are you saying that it's impossible to use the router to navigate to the index? That seems pretty odd

Reply:
I also find it odd but none of these possibilities work:
```
this.$router.push({ name: "home" });
this.$router.push({ name: "index" });
this.$router.push({ path: "/" });
```

All of the above navigate to the not found path nuxt error page.

I think it might be the structure we're using.
Maybe we should be using path `/home/index.vue` for the home page and redirect the  "/" there.

Reply:
let's try to find a better solution for this problem, `window.location` is not a very good solution